src/core/language/service/language.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { CreateLanguageDto } from '../dto/create-language.dto';
+import { UpdateLanguageDto } from '../dto/update-language.dto';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Language } from '../entities/language.entity';
+import { LANGUAGE_NOT_FOUND } from '../messages';
+
+@Injectable()
+export class LanguageService {
+  constructor(
+    @InjectRepository(Language)
+    private readonly languageRepository: Repository<Language>,
+  ) {}
+
+  async create(createLanguageDto: CreateLanguageDto): Promise<Language> {
+    const language = this.languageRepository.create(createLanguageDto);
+    await this.languageRepository.save(language);
+    return language;
```

Comment:
Lo he comentado varias veces e.e
```suggestion
    return  await this.languageRepository.save(createLanguageDto);
```